### ISSUE TITLE
Not being able to manually save

### ISSUE THREAD

**Issue by Westermin** [thegreatstorm/game-ansible#2]

**Comment by thegreatstorm** [comment]
What game?

**Comment by thegreatstorm** [comment]
This must be for valheim. I'm not sure how to do it myself. Since you can't communicate with the server remotely.

**Comment by Westermin** [comment]
Yeah it is indeed valheim, I did however see in the changelog they added a -savedir for dedicated servers, not sure if its the same as saving though.
